packages/renderer/src/App.svelte

```diff
@@ -130,7 +129,6 @@ window.events?.receive('kubernetes-navigation', (args: unknown) => {
     <div class="flex flex-row w-full h-full overflow-hidden">
       <QuickPickInput />
       <CustomPick />
-      <CommandPalette />
```

Comment:
I think this component was only used here, so it should be deleted entirely. (CommandPalette.svelte and tests).